pkg/controller/unifiedpushserver/unifiedpushserver_controller.go

```diff
@@ -197,9 +200,151 @@ func (r *ReconcileUnifiedPushServer) Reconcile(request reconcile.Request) (recon
 		}
 	}
 
+	//Begin AMQ resource reconcile
+	if instance.Spec.UseMessageBroker {
+		//check that address space exists
+		addressSpace := newAddressSpace(instance)
+
+		// Set UnifiedPushServer instance as the owner and controller
+		if err := controllerutil.SetControllerReference(instance, addressSpace, r.scheme); err != nil {
+			return reconcile.Result{}, err
+		}
+
+		foundAddressSpace := &enmassev1beta.AddressSpace{}
+		err = r.client.Get(context.TODO(), types.NamespacedName{Name: addressSpace.Name, Namespace: addressSpace.Namespace}, foundAddressSpace)
+		if err != nil && errors.IsNotFound(err) {
+			reqLogger.Info("Creating a new Address Space", "AddressSpace.Namespace", addressSpace.Namespace, "AddressSpace.Name", addressSpace.Name)
+			err = r.client.Create(context.TODO(), addressSpace)
+			if err != nil {
+				return reconcile.Result{}, err
+			}
+			return reconcile.Result{RequeueAfter: time.Second * 10}, nil
```

Comment:
I'm guessing that this is needed so that we come back in 10 seconds to see if it's ready? If so, could you add a log message above it there as some sort of documentation (so that future me or you knows and isn't worried about it just being an assumption).